File: resilience_eval/benchmarks/java/function_inlining_partial_2.java
public class function_inlining_partial_2 {
void inline_this(int[] input0){
	if(input0 != null){
		input0[0] = 1;
	}
}
void function_inlining_partial(int[] input0){
	inline_this(input0);
}
}

File: resilience_eval/benchmarks/scala/function_inlining_partial_2.scala
class function_inlining_partial_2 {
def inline_this(input0:Array[Int]){
	if(input0 != null){
		input0(0) = 1;
	}
}
def function_inlining_partial(input0:Array[Int]){
	inline_this(input0);
}
}
